api/src/routes/protected/settings.test.ts

```diff
@@ -759,7 +759,8 @@ Happy coding!
           website: 'https://www.freecodecamp.org/',
           twitter: 'https://twitter.com/ossia',
           linkedin: 'https://www.linkedin.com/in/quincylarson',
-          githubProfile: 'https://github.com/QuincyLarson'
+          githubProfile: 'https://bsky.app/profile/quincy.bsky.social',
+          bluesky: ''
```

Comment:
```suggestion
          githubProfile: 'https://github.com/QuincyLarson',
          bluesky: 'https://bsky.app/profile/quincy.bsky.social'
```

It appears you changed the wrong URL. Did you mean to do this?

Reply:
That was not intentional, I apologize for that.  I'll correct it and push the fix. Still getting used to the process. Thanks for catching that and for the feedback!